### PR TITLE
expose manipulatorMouseWrap to settings

### PR DESCRIPTION
**Pull request by amzn-ahmadkrm**
*Added manipulatorMouseWrap to settings from existing functionality in ViewportManipulatorController.cpp 

Signed-off-by: amzn-ahmadkrm <ahmadkrm@amazon.com>

### CODE REVIEW

File: Code/Editor/EditorPreferencesPageViewportManipulator.cpp [hunk 1]
```diff
@@ -96,7 +97,10 @@ void CEditorPreferencesPage_ViewportManipulator::Reflect(AZ::SerializeContext& s
             ->Attribute(AZ::Edit::Attributes::Max, AzToolsFramework::MaxManipulatorViewBaseScale)
             ->DataElement(
                 AZ::Edit::UIHandlers::CheckBox, &Manipulators::m_flipManipulatorAxesTowardsView, "Flip Manipulator Axes Towards View",
-                "Determines whether Planar and Linear Manipulators should switch to face the view (camera) in the Editor");
+                "Determines whether Planar and Linear Manipulators should switch to face the view (camera) in the Editor")
+            ->DataElement(
+                AZ::Edit::UIHandlers::CheckBox, &Manipulators::m_manipulatorMouseWrap, "Manipulator Mouse Wrap",
+                "Determines whether the cursor is going to wrap around the soruce widget.");
```
Comment (hultonha) [inline Code/Editor/EditorPreferencesPageViewportManipulator.cpp:103]: update: `soruce` -> `source` (might also be clearer to just say `viewport`)

File: Code/Framework/AzToolsFramework/AzToolsFramework/Input/QtEventToAzInputMapper.h [hunk 2]
```diff
@@ -38,9 +38,9 @@ namespace AzToolsFramework
                               //!< Internally, this will reset the cursor position after each move event to ensure movement
                               //!< events don't allow the cursor to escape. This can be used for typical camera controls
                               //!< like a dolly or rotation, where mouse movement is important but cursor location is not.
-        CursorModeWrapped,    //!< Flags whether the curser is going to wrap around the soruce widget.
-        CursorModeWrappedX,   //!< Flags whether the curser is going to wrap around the soruce widget only on the left and right side.
-        CursorModeWrappedY    //!< Flags whether the curser is going to wrap around the soruce widget only on the top and bottom side.
+        CursorModeWrapped,    //!< Flags whether the cursor is going to wrap around the soruce widget.
+        CursorModeWrappedX,   //!< Flags whether the cursor is going to wrap around the soruce widget only on the left and right side.
+        CursorModeWrappedY    //!< Flags whether the cursor is going to wrap around the soruce widget only on the top and bottom side.
```
Comment (hultonha) [inline Code/Framework/AzToolsFramework/AzToolsFramework/Input/QtEventToAzInputMapper.h:43]: Good spot! 😅 I spotted this too and had a fix in another PR, you beat me to it 🙂

**Review by hultonha — COMMENTED** [review]
Changes look good to me, just a minor update to a comment to change

**Review by moraaar — DISMISSED** [review]
Looking good :) 
Congrats on your first PR @amzn-ahmadkrm

**Comment by moraaar** [comment]
@amzn-ahmadkrm I just noticed that this PR is merging against the wrong development branch. it's merging against `aws-lumberyard-dev/o3de:development`, but it should be merging to `o3de/o3de:development`. Please redirect the target branch, if it's not possible, then close this PR and open a new one.

**Pull request closed without merging**